app/components/designate-form.js

```diff
@@ -236,8 +236,13 @@ export default Ember.Component.extend({
             loadingView.destroy();
             this.get('router').transitionTo("items.index");
           }
-        })
-        .finally(() => {
+        }).catch((error) => {
+          if(error.status === 422){
+            var errors = Ember.$.parseJSON(error.responseText).errors;
+            this.get("messageBox").alert(errors);
+            this.get('router').transitionTo("items.index");
```

Comment:
Can we add this to callback of mesageBox?

Reply:
Yes Sure 👍